mobile-app/app/screens/AppNavigator/screens/Auctions/screens/PlaceBidScreen.tsx

```diff
@@ -279,82 +382,62 @@ export function PlaceBidScreen(props: Props): JSX.Element {
   );
 }
 
-function BidSummaryCard(props: {
-  displaySymbol: string;
-  collateralDisplaySymbols: string[];
-  blockCount: number;
-  liquidationHeight: number;
-  minNextBid: string;
-  totalAuctionValue: string;
-  onPressFullDetails: () => void;
-  minNextBidInUSD: string;
-}): JSX.Element {
+enum BidAmountButtonTypes {
+  Min = "100%",
+  One = "101%",
+  Two = "102%",
+  Five = "105%",
+}
+interface PercentageAmountButtonProps {
+  onPress: (value: BigNumber, type: BidAmountButtonTypes) => void;
+  bidPercentageAmount: BidAmountButtonTypes;
+  minNextBidInToken: BigNumber;
+  hasBorder: boolean;
+}
+function BidAmountButton({
+  onPress,
+  bidPercentageAmount,
+  minNextBidInToken,
+  hasBorder,
+}: PercentageAmountButtonProps): JSX.Element {
+  const minNextBid = new BigNumber(minNextBidInToken);
+  let value = minNextBidInToken;
+
+  switch (bidPercentageAmount) {
+    case BidAmountButtonTypes.Min:
```

Comment:
let's add a `default:` above this line

Reply:
Our lint rule only allows `default` to be at the bottom, I updated.